docs/platforms/react-native/user-feedback/configuration/index.mdx

```diff
@@ -127,6 +131,43 @@ The following styles are available for customisation.
 | `screenshotThumbnail` | `ImageStyle` | The screenshot thumbnail image style.           |
 | `titleContainer`      | `ViewStyle`  | The title container style.                      |
 
+## Feedback Button Customization
+
+You can customize placement of the feedback button, as well as the fonts and colors.
+
+The example below shows how to customize the bottom margin with the `feedbackIntegration`.
+
+```javascript
+import * as Sentry from "@sentry/react-native";
+
+Sentry.feedbackIntegration({
+  buttonOptions: {
+    styles: {
+      triggerButton: {
+        marginBottom: 75,
+      },
+    },
+  },
+})
```

Comment:
i: We could use a complete init so users don't misunderstand they can call feedbackIntegration outside of Sentry init
```suggestion
Sentry.init({
  integrations: [
    Sentry.feedbackIntegration({
      buttonOptions: {
        styles: {
          triggerButton: {
            marginBottom: 75,
          },
        },
      },
    }),
  ],
});
```